components/scripts/gradle/04-validate-remote-build-caching-ci-ci.sh

```diff
@@ -327,7 +327,7 @@ generate_command_to_repeat_experiment() {
     cmd+=("-f")
   fi
 
-  if [[ "${_arg_debug}" == "on" ]]; then
+  if [[ "${debug_mode}" == "on" ]]; then
```

Comment:
why the name change here?

Reply:
`_arg_x` variables are generated by Argbash which is what we use for parsing command line parameters, displaying help text, etc. One of the first things we do in the scripts is call this [`process_arguments`](https://github.com/gradle/gradle-enterprise-build-validation-scripts/blob/18840cc8647dca8976535e7784919bcb0041bb2c/components/scripts/lib/config.sh#L4) function which maps the Argbash generated variables to our own internal variables. This way we have more control over their naming.

For some reason `_arg_debug` was the only we didn't map and called it directly throughout the codebase. See [this thread](https://github.com/gradle/gradle-enterprise-build-validation-scripts/pull/174#discussion_r1008193504) for additional context.

Reply:
Thanks for the context!